docs/proposals/PROPOSAL_E.md

```diff
@@ -171,10 +171,10 @@ Clients SHALL NOT expect manifests uploaded before the [referrers API](#referrer
 For registries that do not support the `referrers` API, a tag MUST be pushed for any manifest containing a `refers` descriptor with the following syntax:
 
 ```text
-<alg>-<ref>.<hash>.<type>
+ref.<alg>-<ref>.<hash>.<type>
```

Comment:
Do you have an example of another type of prefix you imagine? I'm a little worried about prematurely generalizing here, if we don't know what else will end up using this.

Reply:
I was thinking really about there already being two mechanisms for referrers (oras and the reference types wg), and that this could be a fallback mechanism for either. How would I tell which tag was from where? That is hypothetical.

However, it also helps people who don't know that tags are being used for references have some idea what they're looking at, especially if we take off the type as part of #65.

Reply:
I really genuinely hope that when this working group delivers its changes/recommendations to the OCI specs, that there is no need for ORAS to go off and build their own thing. Many ORAS folks are actively involved in OCI and in this WG, so if they have other needs that aren't met by this, they have a seat at the table to change that.

As for other non-ORAS groups that may want their own digest-tag-style ~~disgusting hack~~ spec enhancement, we can evaluate their proposed solution based on their needs and on the state of registries in the wild at that time. Worst case, they add `:cats.sha256-abcd` as their special namespace, and this WG's type is the "default". But even that I hope never happens.

Reply:
Fully agree that I hope it never happens.

What about the indicator to people wondering "why are there all these tags in my repository that I don't recognise?"

Reply:
I'm not sure that a `ref.` prefix does much to answer that question on its own. A person surprised to find `:sha256-abcd` in their tag list would probably google it and (hopefully, eventually) find a useful answer, just the same as they'd google `[ref.sha256 docker tag]`.

Hopefully even-more-eventually I'd hope registries would adopt support for references and we can be rid of digest tags entirely, some time around 2060. 👴

Reply:
Well yes, this is really attempting to put lipstick on a pig, as the saying goes. Will close this one then.